provider/bidengine/pricing.go

```diff
@@ -246,11 +322,11 @@ func MakeShellScriptPricing(path string, processLimit uint, runtimeLimit time.Du
 }
 
 type dataForScriptElement struct {
-	Memory           uint64 `json:"memory"`
-	CPU              uint64 `json:"cpu"`
-	Storage          uint64 `json:"storage"`
-	Count            uint32 `json:"count"`
-	EndpointQuantity int    `json:"endpoint_quantity"`
+	Memory           uint64            `json:"memory"`
+	CPU              uint64            `json:"cpu"`
+	Storage          map[string]uint64 `json:"storage"`
```

Comment:
This needs to be communicated as a breaking change in the release notes. Otherwise it should be fine.